packages/grafana-data/src/types/datasource.ts

```diff
@@ -293,6 +293,7 @@ export interface QueryEditorProps<
 > {
   datasource: DSType;
   query: TQuery;
+  range: TimeRange;
```

Comment:
To Angular Query Editors, we are passing time range. However, we didn't do this for React query editors. In Explore, we were passing absolute range. I tried to unify this and pass range to Explore Editors, React Editors and Angular Editors, to keep it consistent. We need time range in some editors (Loki) to fetch labels/values based on selected time range.